src/Ormolu/Printer/Meat/Module.hs

```diff
@@ -59,6 +63,26 @@ p_hsModule loc@(L moduleSpan hsModule) = do
     when (trailingComments && isJust hsmodName) newline
     spitRemainingComments
 
+-- | Layout the WARNING/DEPRECATED pragmas in the module head
+
+p_warningTxt :: WarningTxt -> R ()
+p_warningTxt = \case
+  WarningTxt _ lits -> p_pragma "WARNING" lits
+  DeprecatedTxt _ lits -> p_pragma "DEPRECATED" lits
+  where
+    p_pragma :: Text -> [Located StringLiteral] -> R ()
+    p_pragma pragmaText lits = switchLayout (litsSpan lits) $ do
+      breakpoint
+      inci $ pragma (litsSpan lits) pragmaText (p_lits lits)
+
+    litsSpan :: [Located StringLiteral] -> SrcSpan
+    litsSpan lits = combineSrcSpans (getLoc $ head lits) (getLoc $ last lits)
```

Comment:
We might actually be doing this wrong. Maybe we should combine all the `SrcSpan` from the list. We should use:
https://github.com/tweag/ormolu/blob/master/src/Ormolu/Utils.hs#L19

Reply:
Hmm, as long as we know the spans are in order, then it's more efficient to do it the way we're doing it now. But maybe we don't know that for sure :man_shrugging:.

@mrkkrp what do you think?

Reply:
I changed the code to use all spans in #101.